boa/src/builtins/infinity/tests.rs

```diff
@@ -0,0 +1,10 @@
+use crate::exec;
+
+#[test]
```

Comment:
Another test would be to test if `Number.POSITIVE_INFINITY === Infinity` should evaluate to `true`

Reply:
thanks! I've added 
`
#[test]
fn infinity_exists_and_equals_to_number_positive_infinity_value() {
    let scenario = r#"
        Number.POSITIVE_INFINITY === Infinity;
        "#;

    assert_eq!(&exec(scenario), "true");
}`

looks fine?

Reply:
> looks fine?

Yes. Looks good :)